src/mod/endpoints/mod_sofia/sofia_presence.c

```diff
@@ -2987,12 +2987,26 @@ static int sofia_presence_sub_callback(void *pArg, int argc, char **argv, char *
 
 			if (!strcasecmp(astate, "early") || !strcasecmp(astate, "confirmed")) {
 
+				char *cmd;
+				char *callee_id_name = NULL;
+				switch_stream_handle_t user_data_stream = { 0 };
+
 				clean_to_user = switch_mprintf("%s", sub_to_user ? sub_to_user : to_user);
 				clean_from_user = switch_mprintf("%s", from_id ? from_id : from_user);
 
+				SWITCH_STANDARD_STREAM(user_data_stream);
+				cmd = switch_mprintf("%s@%s var effective_caller_id_name", clean_to_user, host);
+				if (switch_api_execute("user_data", cmd, NULL, &user_data_stream) == SWITCH_STATUS_SUCCESS) {
+					if (strncmp("-ERR", user_data_stream.data, 4)) {
+						callee_id_name = (char *)user_data_stream.data;
+					}
+				}
+				switch_safe_free(cmd);
```

Comment:
This line does not follow the coding guidelines. Please see https://github.com/signalwire/libks/blob/master/CODING_GUIDELINES#L36